Servers/Administration/OJS.Servers.Administration/Controllers/BackgroundJobsController.cs

```diff
@@ -51,4 +51,26 @@ public IActionResult AddRecurringDeletingOldSubmissionsForProcessingJob()
 
         return this.Redirect("/");
     }
+
+    [HttpGet]
+    public ActionResult? RegisterJobForUpdatingParticipantTotalScoreSnapshot()
```

Comment:
As per changes merged from yesterday, there is a HangfireJobsHostedService for this. Use the implementation and logic there.